### ISSUE TITLE
Not working with xargs

### ISSUE THREAD

**Issue by sarnobat** [wookayin/python-imgcat#3]
If I try to run this excellent tool inside a command such as this (which I do heavily):

```
find images/ -iname "*dog*.jpg" | xargs -n 1 imgcat
```
you get:
```
error: when reading from stdin, arg should not be given
```
What should happen in this usage model is that `imgcat` should run one arg at a time on each file that matches the glob pattern `*dog*.jpg`.

Is there any chance this can be fixed? Once it is, the tool is perfect :)

**Comment by wookayin** [comment]
Good point. Fixed in ad467da -- please try out the development version until it gets released.

**Comment by sarnobat** [comment]
Works perfectly. This could revolutionize how I use my computer :)
